src/community/release/ext-geofence-server.xml

```diff
@@ -21,7 +21,7 @@
         <include>*geofence-services*</include>
         <include>*geofence-persistence*</include>
         <include>hatbox*</include>
-        <include>*hibernate*</include>
+        <include>hibernate*</include>
```

Comment:
Is this unrelated? Or keycloack brings in hibernate jars that should not be in GeoFence?

Reply:
Exactly. Written like this the geofence will import <evrything>-hibernate which might cause issues to the keycloak plugin if not correctly configured. Typical example is the gs-monitoring-hibernate.jar which is wrongly included into the geofence-server package and therefore causing issues to security plugins too.